### DISCUSSION TITLE
Use uibuilder with Angular

### DISCUSSION THREAD

**Discussion by maddin79** [TotallyInformation/node-red-contrib-uibuilder#198] [Q&A]
Hello, 

first of all, thanks for the work. It is a great library and fun to play with. 

I have one problem. I want to use Angular as front end library but can not get the uibuilder lib working. I'm working Intellij IDEA and I don't know how to include the library, build and use the dist folder in the uibuilder node. I already searched and tried for ours all recommendations how to include a js library into Angular, but nothing works. 
Could someone give me a hint how to do that? I'm also not a pro in Angular or node.js. But I learned Angular a bit and don't want to spent more time in learning another framework. 

Any help is highly appreciated. 

Best Martin

**Comment by TotallyInformation** [comment]
Hi Martin, what versions of Angular, uibuilder, Node-RED and Node.js are you using? Also what does your entry HTML look like?

**Comment by maddin79** [comment]
Hi @TotallyInformation, 

in the end, I switched to React. Unfortunately, I have no time to dig deeper into the problem. I'll close this one. 

Thanks

**Reply by TotallyInformation** [reply]
OK, thanks for letting me know. Thankfully, you are not blocked from uibuilder since REACT will work just fine as well 😁
